2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2047 @@
+------------------------------------------------------------
+
+While a wide range of stakeholders share interest in sustainable
+software, we argue that their individual motivation can differ quite
+significantly:
+
+The **general public** benefits from research which supports the common
+good, in other terms: creates a better world, faster. Tax payers have an
+interest in economical use of their tax money, to which duplicated or flawed
+efforts to create research software – in contrast to software reuse – is
+contrary. A subset of this group may be interested in sustainable, i.e.,
+re-usable and understandable, software as part of citizen science.
+
+**Domain researchers** benefit from better software to do more, better,
+and faster research. Sustainable research software supports this through
+validated functionality (e.g., correct algorithms), the potential for
+reuse, and general availability. Sustainable software also potentially
+simplifies building upon previous research results by re-using the
+involved software to produce additional data, or by extending the
+software’s functionality. In light of recent updates to definitions of
+good research practice (DFG 2019a), sustainable research software also
+allows domain researchers to comply with guidelines and best practices.
+Additionally, using a software that is sustainable enough to establish
+itself as a standard tool in a field signifies inclusion in a research
+community. Less directly, researchers may benefit from the existence of
+sustainable standard tools as they yield standard formats, which in
+themselves facilitate reuse of research data.
+
+**Research Software Engineers (RSEs)** have an intrinsic interest in
+sustainable research software. They create better software for research,
+which enables more and better research. RSEs have an inherent interest
+in developing and working with high quality software, as part of
+professional ethics as well as good research practice. RSEs build their
+reputation on high quality software and software citation (Smith et al.
+2016; Druskat 2019), which will open up new career paths. Finally, for
+RSEs, creating sustainable research software is part of an attractive,
+intellectually challenging, and satisfying work environment.
+
+**Research leaders** as well as **research performing organizations**
+mainly focus on the economic aspects and management of research,
+i.e., available funds, people, and time employed to optimize research
+output. Both need to make sure that their employees continually improve
+their qualification and generate impact to improve their standing in the
+various research communities and ensure continued funding. Overseeing
+and enabling the creation of sustainable research software advances
+their visibility in the field and makes their research endeavors both
+more future-proof, and more easily traceable, reproducible, and
+verifiable and thus more likely to attract additional resources
```

Comment:
Is "verifiable" the correct term here?

Reply:
OALD: "that can be checked to show whether it is true or accurate". Should be OK I guess